config/manager/manager.yaml

```diff
@@ -2,85 +2,30 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    controller-tools.k8s.io: "1.0"
-  name: system #TODO: add the prefix here manually, next version of kustomize won't prefix namespaces.
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: controller-manager-service
-  namespace: system
-  labels:
-    control-plane: controller-manager
-    controller-tools.k8s.io: "1.0"
-spec:
-  selector:
     control-plane: controller-manager
-    controller-tools.k8s.io: "1.0"
-  ports:
-  - port: 443
+  name: system
 ---
 apiVersion: apps/v1
-kind: StatefulSet
+kind: Deployment
 metadata:
   name: controller-manager
   namespace: system
   labels:
     control-plane: controller-manager
```

Comment:
This should likely use a different value, otherwise it will conflict with other kubebuilder projects using the defaults in the same namespace.

Reply:
Agreed.
https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1016